xml/System/Range+OffsetAndLength.xml

```diff
@@ -1,138 +0,0 @@
-<Type Name="Range+OffsetAndLength" FullName="System.Range+OffsetAndLength">
```

Comment:
Yep, it's being removed. https://review.docs.microsoft.com/en-us/dotnet/api/system.range.GetOffsetAndLength?branch=CITest&view=netstandard-2.1 now returns a ValueTuple instead of this new type.